lib/trusona/trusonafication.rb

```diff
@@ -58,6 +58,9 @@ def self.find(trusonafication_id)
     # @param timeout [Int] (30) The max amount of time, in seconds, to wait
     #  for a response from the Trusona API when polling for a Trusonafication
     #  result
+    # @option params [Hash] :custom_fields Optional data to be associated with
+    #  this trusonafication and can be used to constomize any UX elements. Total
```

Comment:
Looks like the word "trusonafication" is capitalized in the lines above and below. This should probably be capitalized for consistency.

Reply:
👍 updated